// ng-process-env/src/onprem-environment/processor.spec.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const processor_1 = require("./processor");
const ANGULAR_JSON_MOCK = {
    projects: {
        testApp: {
            architect: {
                build: {
                    configurations: {
                        testConfig: {
                            fileReplacements: [
                                {
                                    replace: "test/path/to/file/environment.ts",
                                    with: "test/path/to/file/environment.ts"
                                }
                            ]
                        }
                    }
                }
            }
        }
    }
};
describe('processor', () => {
    const options = {
        config: 'my-config-name',
        project: "testApp"
    };
    let processor;
    beforeEach(() => {
        processor = new processor_1.EnvProcessor(options);
    });
    it('Look For Base Json Path In Angular.json', () => {
        let path = processor.lookForBaseJsonPathInNG(ANGULAR_JSON_MOCK);
        expect(path).toBe("test/path/to/file");
    });
});
